### PR TITLE
Added flask server and updated git ignore.

### PR DESCRIPTION
**Pull request by DenisMarchiss**
I created a dummy flask aplication for exemple purposes and i updated gitignore to ignore pycharm specific files

### CODE REVIEW

**Comment by DenisMarchiss** [comment]
[Create a dummy Flask app](https://trello.com/c/EAnPurI6/3-create-a-dummy-flask-app)

File: daily_log/app.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route("/hello")
+def hello():
+    return '<input type="text">'
```
Comment (adrianmoloce) [inline daily_log/app.py]: I don't think this is a good example. Stick to hello world for now.

File: daily_log/app.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route("/hello")
+def hello():
+    return 'Hello world'
```
Comment (adrianmoloce) [inline daily_log/app.py]: Add exclamation point.

Comment (adrianmoloce) [inline daily_log/app.py]: Example which hopefully generates a notification.

File: daily_log/app.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route("/hello")
+def hello():
+    return 'Hello world !'
```
Comment (adrianmoloce) [inline daily_log/app.py]: Please remove the annoying space between hello and the exclamation point.

**Review by adrianmoloce — CHANGES_REQUESTED** [review]

**Review by adrianmoloce — APPROVED** [review]

**Pull request merged**